Add `How advances in technology affected the osu! community` news post (#10659)
ppy/osu-wiki: news/2023/2023-12-12-how-advances-in-technology-affected-the-osu-community.md

While the osu! community has managed to stay unswayed as things such as virtual reality and augmented reality, cloud gaming & AI advancements cropped up due to them being inapplicable to osu!, there were still some things that changed the way osu! is played, or at least changed the view of the most optimal way to play it.

It's time to dive in and see how technology can impact the community, what the results of widespread adoption are, or could have been.

## Graphics tablets (2007)

osu! originally started out as an *[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan)* clone for PC, which was a game that was exclusive to the Nintendo DS, a console that came, and could be used with the included stylus pen. When [peppy](https://osu.ppy.sh/users/2) released osu!, it was only natural for players who already had a graphics tablet to try out osu! with it, and since the get-go it was considered a commodity input device that would more closely replicate the original Nintendo DS experience.

> Playing with a tablet is a completely different ball game compared to the mouse, but rather than being a direct upgrade by eliminating mouse drift with its absolute positioning, it brings up new challenges and questions regarding its ideal usage. Ideal pen grip, tablet area, dragging or hovering play styles have all been detrimental to increasing the learning curve, which is why some players choose to remain on the much more familiar and intuitive mouse.

Author: https://github.com/ppy/osu-wiki/pull/10659/files#r1428978566 applied this, I think the feedback here is minor and somewhat subjective

> Over the years many top players and regular players started switching over from mouse to tablet in order to avoid issues associated with mice, however there have always been top players with both input devices, and they are both considered viable for top-level play, each with its own upsides and downsides.

Reviewer: ```suggestion
Over the years, many top players and regular players started switching over from mouse to tablet in order to avoid issues associated with mice, however there have always been top players with both input devices, and they are both considered viable for top-level play, each with its own upsides and downsides.
```

you mean like one player playing both tablet and mouse or that top players always included some tablet and some mouse players, the sentence would suggest former i think? :thinking:

Author: I meant to say that once tablets were easier to obtain that a lot of top players swapped to it to trade the downsides of the mouse with the downsides of the tablet, but top-level play is achievable with both still

Author: the original sentence sucked let me know if this is better now
https://github.com/ppy/osu-wiki/pull/10659/commits/2f9b7b1f66feda3316da94dd668da811023eb2f8

Reviewer: sounds way better indeed, posted some minor wording suggestion at https://github.com/ppy/osu-wiki/pull/10659#discussion_r1429220780

> A great video by community member [Willy](https://osu.ppy.sh/users/3521482) outlines what's written here and more regarding the usage of tablets as input devices:

Reviewer: ```suggestion
A great video by [Willy](https://osu.ppy.sh/users/3521482) outlines all of above and more regarding the usage of tablets as input devices for osu!:
```

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/V3K3C94fs1o" frameborder="0" allowfullscreen></iframe>
</div>

## Gaming mechanical keyboards (2010)

Mechanical keyboards have been a thing for decades, however after Razer put [gaming and mechanical switches into a compelling package](https://press.razer.com/product-news/the-worlds-first-mechanical-keyboard-designed-for-gaming-launched-at-gamescom-2010/), that launched the chain reaction that resulted in the mechanical keyboard market we know of today.

> By 2010 osu! players were already shifting away from clicking circles with their mouse, to clicking with their Z and X keys, so it would make sense that as these new mechanical keyboards were getting easier and cheaper to adopt into their setups, players would start using them for osu! as well. And while there's nothing stopping anyone from playing osu! with the membrane keyboards, the community at large is suggesting to newer players to purchase a mechanical keyboard, as it provides better feedback when a key is pressed.

Reviewer: ```suggestion
By 2010 osu! players were already shifting away from clicking circles with their mouse to mashing their Zs and Xs, so it would make sense that as these new mechanical keyboards were getting easier and cheaper to adopt into their setups, players would start using them for osu! as well. And while there's nothing stopping anyone from playing osu! with the membrane keyboards, the community at large is guiding the newer players to purchase a mechanical keyboard, as it provides better feedback, which is essential to the rhythm-based gameplay.
```

Author: https://github.com/ppy/osu-wiki/pull/10659/files#r1428978830 combined the two pieces of feedback

> The mass adoption of mechanical keyboards did not only occur in the osu! community, but in the gaming community as a whole, and you'd be hard-pressed to find a gaming setup nowadays without one of these smack-dab in the middle of any gamer's desk.

Reviewer: ```suggestion
The mass adoption of mechanical keyboards did not only occur in the osu! community, but in the gaming community altogether, and you'd be hard-pressed to find a setup without one of these smack-dab in the middle of any gamer's desk nowadays.
```

Author: https://github.com/ppy/osu-wiki/pull/10659/files#r1428978853

## High refresh rate LCD monitors (2012)

> Around mid 2012 first LCD high refresh rate monitors were released with the rates as high as 144hz. Even before LCD monitors were a thing CRT's had the capability of refreshing up to 100hz in most cases (and even higher for high-end models), however with the introduction of high refresh rate LCD panels this technology slowly became much easier to adopt, and is now a staple of any modern gaming setup.

Reviewer: ```suggestion
Around mid-2012, first LCD high refresh rate monitors were released with the rates reaching as high as 144hz. Even before LCD monitors were a thing, CRTs had the capability of refreshing up to 100hz in most cases (and even higher for high-end models), however with the introduction of the high refresh rate LCD panels this technology slowly became much easier to adopt, and is now a staple of any modern gaming setup.
```

Author: https://github.com/ppy/osu-wiki/pull/10659/files#r1428979388 applied this

> As osu! is a game with high amount of moving elements no matter the game mode (as shown below), high refresh rate monitors quickly became the next most-sought-after commodity piece of gear. While being an upgrade in comparison to 60hz monitors, and some people swearing by high refresh rate monitors helping them read higher approach rates & scroll speeds, it did not stop players who could not afford them from rising through the ranks and some of them even becoming top players such as [FGSky](https://osu.ppy.sh/users/2094566), regardless of the refresh rates of their monitor.

Reviewer: ```suggestion
As osu! is a game with high amount of moving elements no matter the game mode (as shown below), high refresh rate monitors quickly became the next most-sought-after commodity piece of gear. While being an objective upgrade to the regular 60hz monitors, it did not stop players who could not afford them from rising through the ranks with some of them reaching real high in the rankings, such as [FGSky](https://osu.ppy.sh/users/2094566).
```

Author: https://github.com/ppy/osu-wiki/pull/10659/files#r1428979516 applied a part of this here

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="soonTM" frameborder="0" allowfullscreen></iframe>
</div>

## Touch screen's rise and fall (2015-2017)

> The earliest mentions of using touch screens as an input device came as soon as the game was released, again due to osu!'s roots as an *Osu! Tatakae! Ouendan* clone. Back then touch screens on PCs were not as prevalent, and most touch screen devices were "Tablet PCs", which were laptops that had a keyboard that can be rotated, folded, or outright detached, along with a touchscreen.

Reviewer: ```suggestion
The earliest mentions of using touch screens as an input device came as soon as the game was released, due to, again, the osu! roots as an *Osu! Tatakae! Ouendan* clone. Back then, the touch screens on PCs were not as prevalent, and most such devices (so-called "Tablet PCs") were just laptops with fancy rotatable and foldable keyboards that could be detached to enchance the fullest of the tappable screen.
```

Author: not sure if i like this let me mull it over

Author: https://github.com/ppy/osu-wiki/pull/10659/commits/5dcbb844d02f73974fbfdc6a7c4189ff7e4be89b

It would take many years for the community to see the true potential of using touch screens in osu!, but starting in 2015 we started seeing glimpses, and eventually the peak of the play style in 2017 with the two players noted below.

### Kuvster (2015)

> In 2015, [Kuvster](https://osu.ppy.sh/users/4896624) had a unique opportunity — snag the pp record under a very specific set of circumstances. Back in the day "TAG4" maps were [approved](/wiki/Beatmap/Category#approved), hence you were able to get pp on them. They weren't playable for a regular player, but if you had a touch screen and were apt with it, you were eligible to grab the pp record, which is what they did:

Reviewer: ```suggestion
In 2015, [Kuvster](https://osu.ppy.sh/users/4896624) had a unique opportunity to snag the pp record under a very specific set of circumstances. Back in the day, the ["TAG4" maps](https://osu.ppy.sh/wiki/Beatmap/TAG) were [approved](/wiki/Beatmap/Category#approved), hence you were able to gain pp by playing them. They weren't really supposed to be played by in a single mode, but if you owned a touch screen and were apt with it, you were eligible to grab the pp record, which is exactly what they did:
```

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/jcHiuNcwsiM" frameborder="0" allowfullscreen></iframe>
</div>

> The 577pp record was short-lived though, as after this all TAG4 maps were exempt from giving pp, and in 2019 TAG4 maps were moved to the [Loved category](/wiki/Beatmap/Category#loved). This however is not the event what "killed" touch screen, its demise would come only 2 years later.

Reviewer: ```suggestion
The 577pp record was short-lived though, as after this all "TAG4" beatmaps were exempt from giving pp, and in 2019 they were moved to the [Loved category](/wiki/Beatmap/Category#loved). This however is not the event that "killed" the touch screens, its demise would come only 2 years later.
```

Author: ill apply this but `the touch screen` would correct as its referring to the play style in its entirety and not like multiple touch screens

### Freedomdiver (2017)

> [Freedomdiver](https://osu.ppy.sh/users/8482062) joined the osu! community in May 2016, and was your average player, up until the moment they realised they can play osu! with the touch screen on their laptop.

Reviewer: ```suggestion
[Freedomdiver](https://osu.ppy.sh/users/8482062) joined the osu! community in May 2016, and started as your average player, up until the moment they realised they can play osu! with the touch screen on their laptop.
```

> They quickly surged through the ranks after realizing that gaining pp by playing jump maps with a touch screen was easier, but not without garnering negative attention from the community. The community had turned against them, using them as a prime example of why touch screen is unfair, but it wouldn't be until November 2017 when freedomdiver decided to push touch screen to its limits, siding with the community and using actions instead of words to call for a change.

Reviewer: ```suggestion
They quickly surged through the ranks after realizing that gaining pp by playing jump maps with a touch screen was simply easier given their point-to-point aim skill requirements. That however, did not just slip through without gathering negative attention from the community. The playerbase had turned against them, using them as a prime example of why touch screen was just unfair due to reasons stated before. However, it wouldn't be until November 2017 when freedomdiver decided to push touch screen to its limits, siding with the community and using actions instead of words to call for a change.
```

The community somewhat sarcastically rallied around freedomdiver during one night in November, as they proceeded to break the pp record 3 times, all on the infamous [Yuima-ru*World TVver.](https://osu.ppy.sh/beatmapsets/459149#osu/994495) map. Hundreds of players spectated freedomdiver as they broke the pp record, and thanks to [CPUGeek](https://osu.ppy.sh/users/4172839) you can experience a snippet of the atmosphere surrounding the play:

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/JCU2ZWRj5gg" frameborder="0" allowfullscreen></iframe>
</div>

> A month later on December 15th, touch screen scores had their pp reduced, touch screen detection was introduced, and a new touch screen "Touch Device" mod was added, which would reduce the amount of pp gained from jumps on any map.

Reviewer: ```suggestion
A month later, on December 15th, touch screen scores had their pp reduced along with an introduction of the brand new "Touch Device" mod, which would reduce the amount of pp gained from jumps on any map when osu! detected touch screen being used as a gameplay device.
```

> A unique usage of touch screen nowadays is using it exclusively as an aiming device, while still tapping with the keyboard. For an example of that, you can look at [bonk](https://osu.ppy.sh/users/4209965) and their [first person live play](https://www.youtube.com/watch?v=PE91EAzRcUo) to see what that looks like.

Reviewer: ```suggestion
Nowadays, a new, unique usage for touch screen was figured. Some players use it exclusively as an aiming device, while still tapping to the beat with their good ol' keyboards. You can look at [bonk](https://osu.ppy.sh/users/4209965) and their [first person live play](https://www.youtube.com/watch?v=PE91EAzRcUo) to see what it looks like.
```

## "Rapid trigger" feature on mechanical keyboards (2021)

> "Rapid trigger" is a new keyboard software feature that's first been implemented by Wooting, a mechanical keyboard manufacturer. Due to its analogue nature made possible by the [Hall effect](https://en.wikipedia.org/wiki/Hall_effect), it allows users to modify the position of any key's actuation point, or go beyond the need for a static actuation point altogether by using rapid trigger. The GIF below visually represents how rapid trigger works.

Reviewer: ```suggestion
"Rapid trigger" is a new keyboard software feature that's first been implemented by Wooting, a mechanical keyboard manufacturer. Due to its analogue nature made possible by the [Hall effect](https://en.wikipedia.org/wiki/Hall_effect), it allows users to modify the position of any key's actuation point, or go beyond the need for a static actuation point altogether by using rapid trigger. The GIF below visually represents how it works.
```

Author: https://github.com/ppy/osu-wiki/pull/10659/files#r1428980278 combined

![](/wiki/shared/news/2023-12-12-how-advances-in-technology-affected-the-osu-community/rapid_trigger_example.gif)

This has caused quite the turmoil amongst the community, as some believe this greatly reduces the skill required to be considered "good at speed", and lowers the skill ceiling quite significantly by allowing players to be less precise with their technique, but still make inputs. The other side of the community argues that this is only beneficial for players who "mash" keys instead of trying to accurately hit notes.

> This is still a hotly debated topic, but if you want to get an opinion of a pro player, look no further than [BTMC](https://osu.ppy.sh/users/3171691)'s video on the rapid trigger feature:

Reviewer: ```suggestion
The arguments are still a topic of countless heated debates, but if you want to get an opinion of a pro player, look no further than [BTMC](https://osu.ppy.sh/users/3171691)'s video on the rapid trigger feature:
```

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/cFvdM_0XuAg" frameborder="0" allowfullscreen></iframe>
</div>

---

> This is a fascinating way to look back through the ages and remind ourselves of a different, simpler time. When the community did not have to worry about the best keyboards and tablets to use to become the best player instantaneously, and just enjoyed osu! for what it is, a rhythm game with an amazing community and one of the largest selections of beat maps in any rhythm game ever.

Reviewer: ```suggestion
This is a fascinating way to look back through the ages and remind ourselves of a different, simpler times. When the community did not have to worry about the best keyboards and tablets to use to become the best player instantaneously, and just enjoyed osu! for what it is, a rhythm game with an amazing community and one of the largest selections of beat maps in any rhythm game ever.
```

Author: i used singular instead of plural so the entire sentence would need to be changed to fit "times"

Reviewer 2: The suggested wording here would be incorrect. I think @venix12 made an oops.

Reviewer: oh yeah, sorry thought i removed that one

---
layout: post

> title: How advances in technology affected the osu! community

Reviewer 3: ```suggestion
title: "How advances in technology affected the osu! community"
```
i think the lack of quotes probably breaks something? better safe than sorry imo

Author: ![image](https://github.com/ppy/osu-wiki/assets/24534022/4e1974be-792e-4ce5-94a8-4d7f20630b7b)
its fine bcs i was getting CI fails when quotes were present

date: 2023-12-12 12:30:00 +0000
---

Let's walk back through the past 15 years of osu!, and how various new technologies affected the game and the community.

![](/wiki/shared/news/2023-12-12-how-advances-in-technology-affected-the-osu-community/banner.jpg)

Every now and then there's a new piece of tech that's slated to change the way games are played, whether it be a new input device, a new way to squeeze more FPS out of your PC, or a whole new way to experience games.

> While the osu! community has managed to stay unswayed as innovations such as virtual reality and augmented reality, cloud gaming & AI advancements cropped up due to them being inapplicable to osu!, there were still some that changed the way the game is played, or at least challenged the views on the most optimal way to play it.

Reviewer 3: ```suggestion
While the osu! community has managed to stay unswayed as innovations such as virtual reality, augmented reality, cloud gaming, and AI advancements cropped up due to them being inapplicable to osu!, there were still some that changed the way the game is played, or at least challenged the views on the most optimal way to play it.
```

> It's time to dive in and see how technology can impact the community, what the results of widespread adoption are, or could have been.

Reviewer 3: ```suggestion
It's time to dive in the history lane and see how technology has impacted the community, what the results of widespread adoption are, or could have been.
```

better aligns with what the newspost is doing

Author: im not sure you dive down a lane, maybe drive down?

Reviewer 3: drive down works

Author: https://github.com/ppy/osu-wiki/pull/10659/commits/df739b94b9237a643ca2bc0ab08974b4ac432a3c i rewrote the entire sentence [as it felt redundant to me](https://discord.com/channels/188630481301012481/218677502141399041/1187169280463212545)

## Graphics tablets (2007)

osu! originally started out as an *[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan)* clone for PC, which was a game that was exclusive to the Nintendo DS, a console that came, and could be used with the included stylus pen. When [peppy](https://osu.ppy.sh/users/2) released osu!, it was only natural for players who already owned a graphics tablet to try it out in the so far unheard-of osu! context, and since the get-go it was considered a commodity input device that would more closely replicate the original Nintendo DS experience.

Playing with a tablet is a completely different ball game compared to the mouse. Rather than being a direct upgrade by eliminating mouse drift with its absolute positioning, it brings up new challenges and questions regarding its optimal usage. A player figuring out their ideal pen grip, tablet area, and dragging or hovering play style have all been detrimental to increasing the learning curve, which is why some players choose to remain on the much more familiar and intuitive mouse.

> Over the years many top players and regular players started switching over to tablet in order to avoid issues associated with mice. Still, there has always been both mouse and tablet top players, as both are considered viable for top-level play, each with its own upsides and downsides.

Reviewer: ```suggestion
Over the years many top players and regular players started switching over to tablet in order to avoid issues associated with mice. Still, there has always been both mouse and tablet players that would compete at the very top of the leaderboards, as both are considered viable for competetive play, each with its own upsides and downsides.
```

Author: ```suggestion
Over the years many top players and regular players started switching over to tablet in order to avoid issues associated with mice. Still, there has always been both mouse and tablet players that would compete at the very top of the leaderboards, as both are considered viable for competitive play, each with its own upsides and downsides.
```
your suggestion is fine but i needed to fix a typo

> Over the years many top players and regular players started switching over to tablet in order to avoid issues associated with mice. Still, there has always been both mouse and tablet players that would compete at the very top of the leaderboards, as both are considered viable for competitive play, each with its own upsides and downsides.

Reviewer 3: ```suggestion
Over the years, many top players and regular players started switching over to tablet in order to avoid issues associated with mice. Still, there has always been both mouse and tablet players that would compete at the very top of the leaderboards, as both are considered viable for competitive play, each with its own upsides and downsides.
```

A great video by [Willy](https://osu.ppy.sh/users/3521482) outlines all of above and more regarding the usage of tablets as input devices for osu!:

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/V3K3C94fs1o" frameborder="0" allowfullscreen></iframe>
</div>

> ## Gaming mechanical keyboards (2010)

Reviewer 3: in this section, do you perhaps want to mention how most mech keyboards have N-key rollover compared to membrane? this logically makes it more favorable to play mania with mech kbs rather than membrane i'd imagine

Author: sure

Author: https://github.com/ppy/osu-wiki/pull/10659/commits/7d42deee50f6fe31cf664ebc0ef20dc18307823d done

Mechanical keyboards have been a thing for decades, however after Razer put [gaming and mechanical switches into a compelling package](https://press.razer.com/product-news/the-worlds-first-mechanical-keyboard-designed-for-gaming-launched-at-gamescom-2010/), that launched the chain reaction that resulted in the mechanical keyboard market we know of today.

> By 2010 osu! players were already shifting away from clicking circles with their mouse, to clicking with their Z and X keys, so it would make sense that as these new mechanical keyboards were getting easier and cheaper to adopt into their setups, players would start using them for osu! as well. And while there's nothing stopping anyone from playing osu! with the membrane keyboards, the community at large is suggesting to newer players to purchase a mechanical keyboard, as it provides better feedback when a key is pressed.

Author: ```suggestion
By 2010, osu! players were already shifting away from clicking circles with their mouse, to mashing with their Zs and Xs keys, so it would make sense that as these new mechanical keyboards were getting easier and cheaper to adopt into their setups. While there's nothing stopping anyone from playing osu! with the membrane keyboards, the community at large is guiding the newer players to purchase a mechanical keyboard, which is essential to the rhythm-based gameplay.
```
combined both suggestions

> By 2010, osu! players were already shifting away from clicking circles with their mouse, to mashing with their Zs and Xs keys, so it would make sense that as these new mechanical keyboards were getting easier and cheaper to adopt into their setups. While there's nothing stopping anyone from playing osu! with the membrane keyboards, the community at large is guiding the newer players to purchase a mechanical keyboard, which is essential to the rhythm-based gameplay.

Reviewer 3: ```suggestion
By 2010, osu! players were already shifting away from clicking circles with their mouse, to mashing with their Z and X keys, so it would make sense that as these new mechanical keyboards were getting easier and cheaper to adopt into their setups. While there's nothing stopping anyone from playing osu! with membrane keyboards, the community at large is guiding the newer players to purchase a mechanical keyboard, which is essential to the rhythm-based gameplay.
```
plural form is already established by `keys`

> The mass adoption of mechanical keyboards did not only occur in the osu! community, but in the gaming community as a whole, and you'd be hard-pressed to find a gaming setup nowadays without one of these smack-dab in the middle of any gamer's desk.

Author: ```suggestion
The mass adoption of mechanical keyboards was not just limited to the osu! community, but in the gaming community as altogether, and you'd be hard-pressed to find a gaming setup without one of these smack-dab in the middle of any gamer's desk nowadays.
```
again combined venix's and your feedback

## High refresh rate LCD monitors (2012)

> Around mid 2012 first LCD high refresh rate monitors were released with the rates as high as 144hz. Even before LCD monitors were a thing CRT's had the capability of refreshing up to 100hz in most cases (and even higher for high-end models), however with the introduction of high refresh rate LCD panels this technology slowly became much easier to adopt, and is now a staple of any modern gaming setup.

Reviewer 4: ```suggestion
Around mid 2012, the first LCD high-refresh rate monitors were released with refresh rates of up to 144 Hz. Even before LCD monitors were a thing, CRT monitors had the capability of refreshing up to 100 Hz in most cases (and even higher for high-end models). However with the introduction of high-refresh rate LCD panels this technology slowly became much easier to adopt, and is now a staple of any modern gaming setup.
```

Remove duplicate use of "high" in first sentence.
[NIT] Hertz should be styled as "100 Hz" (note space and capitalization)

> As osu! is a game with high amount of moving elements no matter the game mode (as shown below), high refresh rate monitors quickly became the next most-sought-after commodity piece of gear. While being an upgrade in comparison to 60hz monitors, and some people swearing by high refresh rate monitors helping them read higher approach rates & scroll speeds, it did not stop players who could not afford them from rising through the ranks and some of them even becoming top players such as [FGSky](https://osu.ppy.sh/users/2094566), regardless of the refresh rates of their monitor.

Reviewer 4: ```suggestion
As osu! is a game with high amount of moving elements no matter the game mode (as shown below), high refresh rate monitors quickly became the next most sought-after commodity piece of gear. While being an upgrade in comparison to common 60hz monitors, and some people swearing by high refresh rate monitors helping them read higher approach rates & scroll speeds, this didn't prevent players who could not afford them from rising through the ranks, and some of them even becoming top players such as [FGSky](https://osu.ppy.sh/users/2094566), regardless of the refresh rates of their monitor.
```

Second sentence is very run-on. Consider splitting somewhere.

Author: ```suggestion
As osu! is a game with high amount of moving elements no matter the game mode (as shown below), high refresh rate monitors quickly became the next most-sought-after commodity piece of gear. While being an objective upgrade to the regular 60 Hz monitors, some people swear by high refresh rate monitors helping them read higher approach rates & scroll speeds. Still, this didn't prevent players who could not afford them from rising through the ranks, and some of them even becoming top players such as [FGSky](https://osu.ppy.sh/users/2094566), regardless of the refresh rates of their monitor.
```

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="soonTM" frameborder="0" allowfullscreen></iframe>
</div>

## Touch screen's rise and fall (2015-2017)

> The earliest mentions of using touch screens as an input device came as soon as the game was released, again due to osu!'s roots as an *Osu! Tatakae! Ouendan* clone. Back then touch screens on PCs were not as prevalent, and most touch screen devices were "Tablet PCs", which were laptops that had a keyboard that can be rotated, folded, or outright detached, along with a touchscreen.

Reviewer 4: ```suggestion
The earliest mentions of using touch screens as an input device came as soon as the game was released, again due to osu!'s roots as an *Osu! Tatakae! Ouendan* clone. Back then, touch screens on PCs were not as prevalent, and most touch screen devices were "Tablet PCs", which were laptops that had a keyboard that can be rotated, folded, or outright detached, along with a touchscreen.
```

> It would take many years for the community to see the true potential of using touch screens in osu!, but starting in 2015 we started seeing glimpses, and eventually the peak of the play style in 2017 with the two players noted below.

Reviewer 4: ```suggestion
It would take many years for the community to see the true potential of using touch screens in osu!, but starting in 2015 we began seeing glimpses of this with Kuvster, and eventually the peak of the play style in 2017 with freedomdiver.
```

Author: ```suggestion
It would take many years for the community to see the true potential of using touch screens in osu!, but starting in 2015 we began seeing glimpses of this with [Kuvster](https://osu.ppy.sh/users/4896624), and eventually the peak of the play style in 2017 with [Freedomdiver](https://osu.ppy.sh/users/8482062).
```
Since it's their first mention I'm gonna have to move where their profiles are linked

### Kuvster (2015)

In 2015, Kuvster had a unique opportunity to snag the pp record under a very specific set of circumstances. Back in the day, ["TAG4" maps](/wiki/Beatmap/TAG) were [approved](/wiki/Beatmap/Category#approved), hence you were able to gain pp by playing them. They weren't really supposed to be played by in a single mode, but if you owned a touch screen and were apt with it, you were eligible to grab the pp record, which is exactly what they did:

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/jcHiuNcwsiM" frameborder="0" allowfullscreen></iframe>
</div>

> The 577pp record was short-lived though, as after this all "TAG4" beat maps were exempt from giving pp, and in 2019 they were moved to the [Loved category](/wiki/Beatmap/Category#loved). This however is not the event that "killed" the touch screens, its demise would come only 2 years later.

Reviewer 3: ```suggestion
The 577pp record was short-lived though, as after this all "TAG4" beat maps were exempt from giving pp, and in 2019 they were moved to the [Loved category](/wiki/Beatmap/Category#loved). This however is not the event that "killed" the touch screen meta, its demise would come only 2 years later.
```

### Freedomdiver (2017)

[Freedomdiver](https://osu.ppy.sh/users/8482062) joined the osu! community in May 2016, and was your average player, up until the moment they realised they can play osu! with the touch screen on their laptop.

> They quickly surged through the ranks after realizing that gaining pp by playing jump maps with a touch screen was easier, but not without garnering negative attention from the community. The community had turned against them, using them as a prime example of why touch screen is unfair, but it wouldn't be until November 2017 when freedomdiver decided to push touch screen to its limits, siding with the community and using actions instead of words to call for a change.

Reviewer 4: ```suggestion
They quickly surged through the ranks after realizing that gaining pp by playing jump maps with a touch screen was easier, but not without garnering negative attention from the community. The community had turned against them, using them as a prime example of why touch screen is unfair. It wouldn't be until November 2017 when freedomdiver decided to push touch screen to its limits, siding with the community and using actions instead of words to call for a change.
```

Author: applied venix's feedback that addresses this as well

The community somewhat sarcastically rallied around freedomdiver during one night in November, as they proceeded to break the pp record 3 times, all on the infamous [Yuima-ru*World TVver.](https://osu.ppy.sh/beatmapsets/459149#osu/994495) map. Hundreds of players spectated freedomdiver as they broke the pp record, and thanks to [CPUGeek](https://osu.ppy.sh/users/4172839) you can experience a snippet of the atmosphere surrounding the play:

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/JCU2ZWRj5gg" frameborder="0" allowfullscreen></iframe>
</div>

> A month later on December 15th, touch screen scores had their pp reduced, touch screen detection was introduced, and a new touch screen "Touch Device" mod was added, which would reduce the amount of pp gained from jumps on any map.

Reviewer 4: ```suggestion
A month later on December 15th, touch screen scores had their pp reduced. Touch screen detection was introduced, and a new touch screen "Touch Device" mod was added, which would reduce the amount of pp gained from jumps on any map.
```

Author: applied venix's feedback on this

A unique usage of touch screen nowadays is using it exclusively as an aiming device, while still tapping with the keyboard. For an example of that, you can look at [bonk](https://osu.ppy.sh/users/4209965) and their [first person live play](https://www.youtube.com/watch?v=PE91EAzRcUo) to see what that looks like.

## "Rapid trigger" feature on mechanical keyboards (2021)

> "Rapid trigger" is a new keyboard software feature that's first been implemented by Wooting, a mechanical keyboard manufacturer. Due to its analogue nature made possible by the [Hall effect](https://en.wikipedia.org/wiki/Hall_effect), it allows users to modify the position of any key's actuation point, or go beyond the need for a static actuation point altogether by using rapid trigger. The GIF below visually represents how rapid trigger works.

Reviewer 4: ```suggestion
"Rapid trigger" is a new keyboard software feature that's first been implemented by Wooting, a mechanical keyboard manufacturer. Due to its analogue nature made possible by the [Hall effect](https://en.wikipedia.org/wiki/Hall_effect), it allows users to modify the position of any key's actuation point, or go beyond the need for a static actuation point altogether by using rapid trigger. The animation below visually represents how rapid trigger works:
```

Author: ```suggestion
"Rapid trigger" is a new keyboard software feature that's first been implemented by Wooting, a mechanical keyboard manufacturer. Due to its analogue nature made possible by the [Hall effect](https://en.wikipedia.org/wiki/Hall_effect), it allows users to modify the position of any key's actuation point, or go beyond the need for a static actuation point altogether by using rapid trigger. The animation below visually represents how it works.
```

combined venix and your feedback

![](/wiki/shared/news/2023-12-12-how-advances-in-technology-affected-the-osu-community/rapid_trigger_example.gif)

This has caused quite the turmoil amongst the community, as some believe this greatly reduces the skill required to be considered "good at speed", and lowers the skill ceiling quite significantly by allowing players to be less precise with their technique, but still make inputs. The other side of the community argues that this is only beneficial for players who "mash" keys instead of trying to accurately hit notes.

The arguments are still a topic of countless heated debates, but if you want to get an opinion of a pro player, look no further than [BTMC](https://osu.ppy.sh/users/3171691)'s video on the rapid trigger feature:

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/cFvdM_0XuAg" frameborder="0" allowfullscreen></iframe>
</div>

---

> This is a fascinating way to look back through the ages and remind ourselves of a different, simpler time. When the community did not have to worry about the best keyboards and tablets to use to become the best player instantaneously, and just enjoyed osu! for what it is, a rhythm game with an amazing community and one of the largest selections of beat maps in any rhythm game ever.

Reviewer 3: ```suggestion
This is a fascinating way to look back through the ages and remind ourselves of a different, simpler time. When the community did not have to worry about the best keyboards and tablets to use to become the best player instantaneously, and just enjoyed osu! for what it is, a rhythm game with an amazing community and one of the largest selections of beatmaps/charts in any rhythm game ever.
```
